framework/include/cppmicroservices/LDAPFilter.h

```diff
@@ -68,7 +68,7 @@ class US_Framework_EXPORT LDAPFilter
 {
 
 private:
-  typedef SharedDataPointer<LDAPFilterData> LDAPFilter::*bool_type;
+  using bool_type = SharedDataPointer<LDAPFilterData> LDAPFilter::*;
```

Comment:
The whole line is a left-over and can be deleted.

Reply:
Done.